component/web/src/main/java/io/meeds/social/handler/SpaceCreationInstantiationHandler.java

```diff
@@ -112,7 +112,7 @@ public boolean execute(ControllerContext controllerContext) throws Exception {
         saveSpaceAvatar(space, model);
         saveSpaceBanner(model, space);
         removeTokenCookie(request, response);
-        String path = servletContext.getContextPath() + "/s/" + space.getSpaceId();
+        String path = servletContext.getContextPath() + "/spaces/" + space.getSpaceId();
```

Comment:
`/portal/spaces/155876` doesn't exist I think, only `/portal/s/155876` can redirect to the space Home page

```suggestion
        String path = servletContext.getContextPath() + "/s/" + space.getSpaceId();
```